python/lsst/cp/pipe/ptc.py

```diff
@@ -705,6 +705,11 @@ def fitPtcAndNonLinearity(self, dataset, tableArray, ptcFitType):
             'ASTIERAPPROXIMATION' to the PTC
         tableArray : `np.array`
             Look-up table array with size rows=nAmps and columns=ADU values
+
+        Returns
+        -------
+        dataset : `lsst.cp.pipe.ptc.PhotonTransferCurveDataset`
```

Comment:
This `dataset` is the same object as the `dataset` parameter, but has been modified (right?). Let's document that — rather than just repeating the same words, let's be explicit about what this method changes in that object.

Reply:
I have expanded the docstring with clarifications.